### PR TITLE
[9/3/2022] Team-2 Add Education

### PR DESCRIPTION
**Pull request by ajf876**
Added an experience feature for cucumber

### CODE REVIEW

File: src/test/resources/features/addEducation.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@ui @regression
+Feature: Add Education
+
+  Scenario: Title of your scenario
+    Given user is logged in
+      | username                     | password   |
+      | yadikaeryashengdmv@gmail.com | yadang0306 |
+    Then User updates "add education"
```
Comment (Muradil-Erkin) [inline src/test/resources/features/addEducation.feature]: All the `Education` data should come in from the feature file

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

File: src/test/java/uiStepDefinitions/AddEducation.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		driver.findElement(By.name("from")).sendKeys("05/07/2022");
+		driver.findElement(By.name("current")).click();
+		driver.findElement(By.tagName("textarea")).sendKeys("This is fun");
+		driver.findElement(By.xpath("//input[@type='submit']")).click();
+		Thread.sleep(1000);
+
+	}
+
+	@And("user should see the success alert for add education")
+	public void user_should_see_the_success_alert_works() {
```
Comment (Muradil-Erkin) [inline src/test/java/uiStepDefinitions/AddEducation.java]: There's already a step that does this validation, please remove this and use the existing one.

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

File: src/test/java/uiStepDefinitions/AddEducation.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import utilities.DriverFactory;
+
+public class AddEducation {
+
+	private WebDriver driver = DriverFactory.getInstance();
+
+	@Then("User updates {string}")
+	public void user_updates(String string, io.cucumber.datatable.DataTable dataTable) throws InterruptedException {
```
Comment (Muradil-Erkin) [inline src/test/java/uiStepDefinitions/AddEducation.java]: - The step doesn't make sense, it should say something like `user adds an education credential`
- There should be just 1 data table as a parameter, not sure about that String

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]
I'm sure the test doesn't run now

File: src/test/resources/features/addEducation.feature [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+      | school   | degree                 | fieldofstudy  | from date  | current | to   | description  |
+      | BoraTech | Test automation degree | Test Automation | 05/07/2022 | true    | null | This is fun! |
```
Comment (Muradil-Erkin) [inline src/test/resources/features/addEducation.feature:10]: Data table is not formatted nicely

File: src/test/resources/features/addEducation.feature [hunk 5]
```diff
@@ -0,0 +1,11 @@
+@ui @regression
+Feature: Add Education
+
+  Scenario: Title of your scenario
```
Comment (Muradil-Erkin) [inline src/test/resources/features/addEducation.feature]: Tag the scenario, not the feature.
And the title of scenario doesn't make sense.

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Review by Muradil-Erkin — APPROVED** [review]
LGTM!

**Review by yadikaer — APPROVED** [review]

**Pull request merged**